composer.json

```diff
@@ -20,7 +20,7 @@
     "license": "MIT",
     "require": {
         "php": ">=5.4",
-        "firebase/php-jwt": "^3.0"
+        "firebase/php-jwt": "^4.0"
```

Comment:
I'd like to suggest using `firebase/php-jwt": "^3.0|^4.0"` to stay backwards compatible - otherwise this change would require a new major release of the token generator.